internal/manifests/manifestutils/params.go

```diff
@@ -38,11 +38,25 @@ type GCS struct {
 
 // S3 holds S3 configuration.
 type S3 struct {
+	LongLived  *S3LongLived
+	ShortLived *S3ShortLived
+}
+
+// S3LongLived holds long-lived S3 configuration.
```

Comment:
Can you add a little bit more of info about what it means `long-lived` vs `short-lived` S3?